src/assets/i18n/sv.json

```diff
@@ -183,7 +183,30 @@
     "OBSERVATIONS": "Observationer"
   },
   "OFFLINE_MAP": {
-    "TITLE": "Offline kartor"
+    "OFFLINE_MAP_PAGE_TITLE": "Offline kartor"
+  },
```

Comment:
Er tekstene delt opp i to objekter her? Kan være det fører til at det ene eller andre objektet forsvinner, evt at jsonen blir ugyldig.